cli/command/service/update.go

```diff
@@ -1025,8 +1025,11 @@ func updateNetworks(ctx context.Context, apiClient client.NetworkAPIClient, flag
 	}
 
 	if flags.Changed(flagNetworkAdd) {
-		values := flags.Lookup(flagNetworkAdd).Value.(*opts.ListOpts).GetAll()
-		networks, err := convertNetworks(ctx, apiClient, values)
+		var values *opts.NetworkOpt
+		if values = flags.Lookup(flagNetworkAdd).Value.(*opts.NetworkOpt); values == nil {
```

Comment:
Seems like the `nil` check should happen before the cast, otherwise the cast would panic. But I don't think `Value` can ever be nil here. I would remove the check.

```go
values := flags.Lookup(flagNetworkAdd).Value.(*opts.NetworkOpt)
```